src/lib/types.ts

```diff
@@ -820,6 +820,8 @@ export type AuthMFAEnrollResponse =
            * to use it. Avoid loggin this value to the console. */
           uri: string
         }
+        /** Friendly Name of the Factor **/
```

Comment:
Can you use consistent casing?

```suggestion
        /** Friendly name of the factor, useful for distinguishing between factors **/
```